app/renderer/settings.js

```diff
@@ -315,6 +323,10 @@ const numberRanges = {
     "timeoutlen": [0, 9000000000000000]
 }
 let customStyling = ""
+const bookmarkSettings = [
```

Comment:
You don't need this list, just check for the `bookmarksfile` setting and call the function directly.